src/Domain/SessionConfigJSONSchema.json

```diff
@@ -98,7 +98,14 @@
                 "type": "number"
               },
               "plan_type": {
-                "type": "string"
+                "oneOf": [
```

Comment:
we accept 2 formats for plan_type. The old style still which is x,x,x. e.g. 0,0,0 or 0,1,0 etc
Or just an int which is then processed as bitwise flags